packages/blade/src/components/Carousel/_decisions/decisions.md

```diff
@@ -0,0 +1,175 @@
+| onChange                 | `(slideIndex: number) => void`         | `undefined`                              | Callback which gets fired everytime a slide changes, useful for analytic events (only runs on user interaction not on autoPlay)                                                                              |          |
+| accessibilityLabel       | `string`                               | `undefined`                              | Accessibility label for the carousel, this will let screen reader users know what content the carousel holds (eg: "Product carousel")                                                                        |          |
+
+- Additionally we will also expose few styled props, namely margin/width etc so that users can add negative margins to create end to end bleed carousel.
+- To let consumers scroll to any specific slide we will expose a `ref` with `goToSlide` method.
+
+### CarouselItem
+
+- N/A - CarouselItem will just be a wrapper and have internal styling
+
+## Basic Usage Example
+
+```jsx
+const Testimonial = () => {
+  return (
+    <Box display="flex" gap="spacing.2">
+      <Box>
+        <Title>Acquire Customers From New Customer Segments</Title>
+        <Text weight="bold">Subham Kumar</Text>
+      </Box>
+      <Img src="./avatar-1.png" />
+    </Box>
+  )
+};
+
+<Carousel
+  autoPlay
+  visibleItems={2}
+  navigationButtonPosition="bottom"
+>
+  <CarouselItem>
+    <Testimonial />
+  </CarouselItem>
+  <CarouselItem>
+    <Testimonial />
+  </CarouselItem>
+  <CarouselItem>
+    <Testimonial />
+  </CarouselItem>
+</Carousel>
+```
+
+<img src="./example-usage-1.png" width="70%" />
+
+
+
+## Variants
+
+Carousel has two variants which can solve most of the general & landing page usecases: 
+
+- Variant 1: Without visibleItems
+- Variant 2: With visibleItems.
+
+### **Variant 1: Without visibleItems:**
+
+When the `visibleItems` is set to `autofit`, the carousel will automatically adjust and fit however many carouselItems it can in the available space & it can have: 
+
+- Automatic bleed support 
+- Scroll Overlay support
+- Have `shouldAddStartEndMargin` prop which can be used to replicate [these](https://razorpay.com/gst-number-search/pan/) kind of carousels 
+  <img src="./gst-page-carousel.png" width="50%" />
+
+
+> NOTE: To ensure bleed happens you need to specify a fixed amount of carousel item's width via the `carouselItemWidth` prop. Otherwise the carousel item will take up the 100% of the container width and won't bleed
+
+
+v1: visibleItems: autofit + auto bleed + shouldAddStartEndMargin
+
+<img src="./example-visibleitems-undefined-with-margin.png" width="70%" />
+
+v1: visibleItems: autofit + auto bleed
+
+<img src="./example-visibleitems-undefined-without-margin.png" width="70%" />
```

Comment:
finally will we show the indicators here? I think we decided we'll hide it? can we update the image?

Reply:
Outdated screenshot, we won't show indicators here. will update.

Reply:
Updated and added note about the indicator